metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -130,6 +131,14 @@ def extract_dbt_entities(
         if "identifier" in node and not load_catalog:
             name = node["identifier"]
 
+        comment = key
```

Comment:
Should `key` be used as the fallback? Not sure myself, just wondering why this was chosen when the description is usually left empty when a clear candidate is unavailable.

Reply:
`key` is already being used when setting the `DatasetPropertiesClass.description`. This way the default behavior stays unchanged when `all_catalog_entities[key]["metadata"]["comment"]` is not defined.

Reply:
Got it, make sense